src/core/core.scale.js

```diff
@@ -44,7 +44,7 @@ module.exports = function(Chart) {
 			autoSkipPadding: 0,
 			labelOffset: 0,
 			callback: function(value) {
-				return '' + value;
+				return helpers.isArray(value) ? value : '' + value;
```

Comment:
Might be a good idea to drop a comment here as to why this check needs to happen